internal/occ/cmd/component/exec_test.go

```
@@ -0,0 +1,108 @@
+		{
+			name:        "component with multi-word command after --",
+			args:        []string{"my-service", "--", "echo", "hello", "world"},
+			wantCompArg: "my-service",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			cmd := newExecCmd(func() (client.Interface, error) { return nil, nil })
+			cmd.PreRunE = nil // skip auth check in tests
+			cmd.RunE = nil   // prevent execution
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)